.drone.yml

```diff
@@ -182,18 +200,10 @@ matrix:
   include:
     # phan
     - PHP_VERSION: 7.2
-      TEST_SUITE: phan
-
-    # phpstan
-    - PHP_VERSION: 7.2
-      TEST_SUITE: phpstan
```

Comment:
This tried to run ``phpstan``. See if it worked, if so then keep it.

Reply:
It didn't. I added that myself but had not been configured.

Reply:
This would run this: https://drone.owncloud.com/owncloud/richdocuments/49/21